esx_service/vmci/vmci_client.c

```diff
@@ -51,6 +52,8 @@ typedef struct be_request {
 } be_request;
 
 #define MAXBUF 1024 * 1024 // Safety limit. We do not expect json string > 1M
+#define MAX_CLIENT_PORT 1023 // Last privileged port
+#define START_CLIENT_PORT 1000 // Where to start client port
```

Comment:
It is a brown range.. I could have started from 1 and work up to 1023 but felt that was excessive, also, if we run into the issue with a range of this size, maybe we need to investigate the VMCI performance with the VMCI folks. I can see why a range of 1 can lead to issues but a range of 23 should be large enough. Like I said a brown range...